src/pages/waiting-room-to-car/waiting-room-to-car.ts

```diff
@@ -255,4 +261,7 @@ export class WaitingRoomToCarPage extends BasePageComponent{
     this.form.value['eyesightCtrl'] = '';
     this.store$.dispatch(new EyesightResultReset());
   }
+
+  tellMeQuestionChanged(e): void {
```

Comment:
What's up with this empty function?

Reply:
Was just proving we could hook into the selected question. It will be filled in in the next PR.